projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -56,22 +56,25 @@ Character descriptions can be provided for a locale in a file named characterDes
 This is a UTF-8 encoded text file.
 Blank lines and lines beginning with a "#" character are ignored.
 All other lines should contain a character, followed by a tab, then one or more descriptions separated by tabs.
+Multiple descriptions for a character will be read with natural pauses between them.

```

Comment:
Not exactly. My understanding (with tests) is that reading one character (e.g. numpad2, left/right arrows, pavnum1/3) read all the descriptions separated by a pause. But when many characters are read successively in spelling commands (e.g. triple press of numpad5, numpad8 or NVDA+upArrow), only the first description is used.

@cary-rowen, @hwf1324 could you explain the use case of multiple descriptions in the zh_CN file? Why one is not enough?

Reply:
Yes, one character description is not enough in Chinese, currently our first description is a short description of the character, which serves as a quick way to determine the meaning of the Chinese character, while the second description is a detailed explanation of the exact meaning of the character and examples. Unlike alphabets, Chinese characters are ideographs, and there may be very rich and large amount of information in a single character.

Of course as you described in your comment, this is currently very important when reading multiple kanji in a row, otherwise the spelling would become very lengthy.

Although there are multiple representations of a kanji in the long description, we also seem to be using only two descriptions at the moment (a short description and a long description). For the line you commented on, I don't seem to feel a pause between the two descriptions when reading the description of a kanji.

Sorry for the possible wording bias in my explanation due to the use of translations.

Reply:
How does this read?

```suggestion
Multiple descriptions for a character will be read with natural pauses between them when reading a single character.
When reading character descriptions of multiple subsequent characters, the first description is used for each character.
```